website/docs/guides/spawn.md

```diff
@@ -109,8 +109,8 @@ import { main, spawn } from 'effection';
 import { fetchWeekDay } from './fetch-week-day';
 
 main(function*() {
-  let dayUS = yield spawn(fetchWeekDay('est'));
-  let daySweden = yield spawn(fetchWeekDay('cet'));
+  let dayUS = spawn(fetchWeekDay('est'));
+  let daySweden = spawn(fetchWeekDay('cet'));
```

Comment:
Operations don't actually do anything until they are evaluated, and in order to do that, they must be yielded to. 

`spawn()` is an "higher order" operation. It takes an operation and returns an operation that evaluates that operation concurrently. But, like every other operation, you must yield it.

For example:

```ts
// this just defines the operation, but has no state, and no side-effects
let spawnGetWeekDayCET: Operation<Task<Response>> = spawn(fetchWeekDay('cet'));

// start two separate tasks each which are doing the same operation.
let task1 = yield spawnGetWeekDayCET;
let task2 = yield spawnGetWeekDayCET;
```
So the spawn is definitely necessary here.


> Note: In Effection, side effects happen at yield points. and the code between yield points is generally side-effect free.

Reply:
So the `yield spawn` produce a task, and when we want to get a value from that task we need to `yield task`? And `dayUS` is actually not a value, it's a task, isn't it?

Reply:
in your example:

```ts
let dayUS = spawn(fetchWeekDay('est'));
```

`dayUS` is an `Operation`, not a `Task`. In order to evaluate an operation you `yield`.  Internally, this will create a task to "run" the operation, and then wait until the task is complete. `spawn` takes an operation and returns a _new_ operation that evaluates to a task running the original operation A.

Reply:
Oh, now I understand. Thanks

Reply:
If you can fix the other portions of the PR, I say let's get it merged!